scripts/create_build_adoc_include.py

```diff
@@ -48,6 +49,13 @@ def check_no_markdown(filename):
                     seen_header = True
                     if github_edit is not None:
                         line += edit_text + "\n\n"
+            else:
+                # find all image references, append md5 hash at end to bust the cache if we change the image
+                m = re.match(r'^(image::)(.+)(\[(.+)]\n?)$', line)
```

Comment:
@nathan-contino Do we need to use non-greedy matching here? E.g. `.+?` instead of `.+`? Since it's just one line, it's pretty low risk, but I always get nervous about this stuff. But also not sure if that would change the functionality at all...

Reply:
I don't think it's valid syntax to put things on a line after an image other than the image itself, and most images should have the square brackets. So we can probably get away without worrying. I couldn't find a less-greedy approach that worked with all images, since names vary a lot!